src/App.js

```diff
@@ -1,9 +1,26 @@
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import './App.css';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import BigView from './components/BigView';
+import { getFinance } from './redux/dataSlice/dataSlice';
+import DetailViewItem from './components/DetailViewItem';
 
 function App() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getFinance());
+  }, [dispatch]);
   return (
     <div>
-      
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<BigView />} />
+
+          <Route path="/detail" element={<DetailViewItem />} />
+
```

Comment:
There's one route missing which will direct the user to the details page with a unique ID.

### For Example:
```
<Route exact path="/item/:id" component={Item} />
```

### This is what it shows currently when we visit a details page
<img width="179" alt="Screenshot 2023-06-30 at 1 45 17 AM" src="https://github.com/francksefu/capstone-react/assets/83048208/3cb94382-290e-4c83-8274-9c51054b6e04">

Reply:
This change has been done well